charts/plgd-hub/values.yaml

```diff
@@ -712,20 +712,36 @@ coapgateway:
   # -- Annotations for coap-gateway pod
   podAnnotations: { }
   service:
-    # -- Service type
-    type: LoadBalancer
```

Comment:
This is a breaking change. CoAP Gateway by default doesn't go through the LoadBalancer, so users need to configure the deployment as documented here:
https://docs.plgd.dev/docs/deployment/k8s/#nodeport-for-coap-gateway
This won't work anymore and update of deployment will stop working. 

You don't need to configure the `nodeport` per protocol. Nor the `type`. I would keep the old configuration for the service, (don't change `values.yaml` at all) and create `service-tcp.yaml` if tcp is in protocols, create `service-udp.yaml` if it is listed in protocols, but reuse same `labels`, `annotation`, `node port` as well as `type`.

What is default? TCP or

Reply:
Ok so I suggest to consider `service` at TCP setup and creates `serviceUdp` - this will be backward compatible.